style.css

```diff
@@ -284,6 +284,81 @@ header a {
   transform: scale(1.1);
 }
 
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  width: 100%;
+  background-image: url(/images/contact-form-m-bg.svg);
+  background-size: contain;
+  background-position: top right;
+  background-repeat: no-repeat;
+  padding: 50px 24px;
+}
+
+.contact-section h2 {
+  color: #fff;
+  width: 90%;
+  text-align: center;
+  padding-top: 25px;
+  font-weight: 700px;
+  font-size: 40px;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: -10px;
+  padding: 30px;
+}
+
+.contact-section form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contact-input {
+  padding: 15px;
+  width: 375px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+  outline: none;
+}
+
+.text-area {
+  height: 170px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+.btn-contact {
+  width: 140px;
+  border: none;
+  padding: 12px 16px;
+  background: #fff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  margin-left: 0;
+}
+
 .btn:enabled {
   color: #6070ff;
   background-color: #fff;
```

Comment:
Please consider reviewing your styles. Try using the following syntax to achieve that the pressed functionality:-

`.btn:active{
  color: #FFFFFF;
  background: #2230D2;
}`